### DISCUSSION TITLE
Download release assets on a private project

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#53336] [Mobile]
### Select Topic Area

Bug

### Body

I have a private project, on the following project I released a release that contains assets.

When I go from the app to the project, I see the various releases that are there.
I click on a release and it shows me the assets that are there, when I try to download an asset from download, the default browser window opens (I think) and gives me a 404 error.

Would it be possible that instead of opening the browser, the application would allow you to download the assets normally?

![Screenshot_20230420-205421~2.png](https://user-images.githubusercontent.com/20476002/233461781-e13a0ab7-98b2-4bc5-81e5-2e373ab1e180.png)

![Screenshot_20230420-205446~2.png](https://user-images.githubusercontent.com/20476002/233461803-a118040b-b68f-4cc7-a331-cafadff47809.png)

**Comment by taki-on** [comment]
Thank you very much for reaching out to us. It is probably because those files are private but you do not have permission to get them in the web browser. A workaround would be:

1. Sign in in the default web browser in advance
2. Tap on the files in the app
3. Open the files in the default web browser you signed in already

Then probably you can download them onto your device.

We know that it is inconvenient and we are really sorry about it. This issue is known to us and it is on our list. Please stay tuned!

Thank you!

**Reply by Angelk90** [reply]
@taki-on : is there anything new?
